src/auth/jwtService.ts

```diff
@@ -0,0 +1,33 @@
+import * as jwt from 'jsonwebtoken';
```

Comment:
wrap functions inside singleton `JwtService` also provide injection function for it